quil-rs/src/program/snapshots/quil_rs__program__tests__wrap_in_loop.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: quil-rs/src/program/mod.rs
+expression: program.to_quil().unwrap()
+---
+DECLARE ro BIT[1]
```

Comment:
An idea for the future: we could use `libquil-sys` to validate the behaviour of stuff like this, rather than eye-balling the logic.